slovak/README.md

```diff
@@ -0,0 +1,9 @@
+# Slovak Braille Specification
+
+## [Pravidlá písania a používania Braillovho písma
+v Slovenskej republike](pravidla-bp01.pdf)
```

Comment:
There seems to be a new line because "písma" and "v". Not sure if that matters.

Reply:
Fixed